tests/fltest_screen.fish

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env fish
+# -*-  mode:fish; tab-width:4  -*-
+
+function fltest_screen
+
+    echo "Context: STY var is set to 'fishline_test'"
```

Comment:
A context line should appear before each call to `__fishline_test`, here:
- STY var is set to '76230.test123' (custom session name)
- STY var is set to '69359.ttys001.Providence' (automated OSX style name)
- STY var is set to '20424.pts-0.userctl' (automated Linux style name)